<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="Vinays-MacBook-Air.local" failures="2" tests="4" name="PetTests" time="11.65" errors="0" timestamp="2025-04-26T18:27:19 IST">
  <testcase classname="api.test.UserTests" name="testPostUser" time="4.205"/>
  <testcase classname="api.test.UserTests" name="testGetUserByName" time="2.082"/>
  <testcase classname="api.test.UserTests" name="testUpdateUserByName" time="3.472">
    <failure type="java.lang.AssertionError" message="expected:&amp;lt;404&amp;gt; but was:&amp;lt;200&amp;gt;">
      <![CDATA[java.lang.AssertionError: expected:<404> but was:<200>
at api.test.UserTests.testUpdateUserByName(UserTests.java:72)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
... Removed 30 stack frames]]>
    </failure>
  </testcase> <!-- testUpdateUserByName -->
  <testcase classname="api.test.UserTests" name="testDeleteUserByName" time="1.669">
    <failure type="java.lang.AssertionError" message="expected:&amp;lt;404&amp;gt; but was:&amp;lt;200&amp;gt;">
      <![CDATA[java.lang.AssertionError: expected:<404> but was:<200>
at api.test.UserTests.testDeleteUserByName(UserTests.java:80)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
... Removed 30 stack frames]]>
    </failure>
  </testcase> <!-- testDeleteUserByName -->
</testsuite> <!-- PetTests -->
